### PR TITLE
correct MFSP calculation

### PR DESCRIPTION
**Pull request by jiananf2**
corrected MFSP to $/GGE

### CODE REVIEW

**Comment by yalinli2** [comment]
thanks @jiananf2 ! the tests failed but it's just we need to update the numbers in the [tests](https://github.com/QSD-Group/EXPOsan/blob/main/tests/test_htl.py)

actually, it'll be good if you can run the tests locally, can you try setting up the local testing following these instructions:
https://qsdsan.readthedocs.io/en/latest/CONTRIBUTING.html#testing

if you manage to get it work, I can give you the write access to the `main` branch so you can directly push (after making sure the local tests can pass) and don't need to open PR, thanks!

**Comment by jiananf2** [comment]
Will work on that and I am going to close this PR, thanks!

**Comment by jiananf2** [comment]
Hi @yalinli2, the local testing has passed!

**Comment by yalinli2** [comment]
thanks @jiananf2 for trying the local tests out! but seems like there are conflicts between `main` and `pfas`, also the [online tests](https://github.com/QSD-Group/EXPOsan/actions/runs/3933454865/jobs/6727167524) have failed, can you look into it before merging?

the online tests can fail for a bunch of weird reasons and quite often it's about configuration rather than bugs... let me know if you can't get it work, thanks!

**Review by yalinli2 — APPROVED** [review]

**Comment by yalinli2** [comment]
@jiananf2 did you test with the changes reverted back as this? 
https://github.com/BioSTEAMDevelopmentGroup/thermosteam/commit/7b4468929a654f0ea78b3fcfdc80da659ee2b926#comments

i.e., take out the `try... except...` block and just use 
```python
    x = flx.wegstein(x_iter, x_guess, 1e-10, args=args, checkiter=False,
                     checkconvergence=False, convergenceiter=3)
```

the same error showed up when I did this locally...

thanks!

**Comment by jiananf2** [comment]
I did with the changes in thermosteam. I made change in QSDsan and EXPOsan. Did you use pfas branches in both?
________________________________

**Comment by yalinli2** [comment]
ohhhh I didn't know you made updates in QSDsan, let me try again, thanks!
